### ISSUE TITLE
Add additional arguments to the launch_rocky method to handle existing sessions

### ISSUE THREAD

**Issue by viniciusdaroz** [ansys/pyrocky#103]
### 📝 Description of the feature

When an open session crashes for some reason and the user tries to launch a new one, pyRocky will raise an error saying that there is already an existing session in place. The user is then forced to connect to the existing session using the connect_to_rocky method and close it. It would be nice to have an additional argument under the launch_rocky method to do that for the user. Something like launch_rocky(terminate_existing_session=True). 

Another option would be to deploy the new session to a different port instead of always sticking to the default one. This would be interesting when launching more than one session in parallel is necessary. Maybe an argument such as launch_rocky(allow_multiple_sessions=True) and corresponding close_all_sessions() or close(port=xxxxx) methods.
